systemverilog-plugin/UhdmAst.cc

```diff
@@ -2264,16 +2264,93 @@ void UhdmAst::process_cont_assign()
 
 void UhdmAst::process_assignment()
 {
+    const uhdm_handle *const handle = (const uhdm_handle *)obj_h;
+    const UHDM::BaseClass *const object = (const UHDM::BaseClass *)handle->object;
```

Comment:
It's already done in `process_object`, we can just pass it to `process_assignment` like in other functions.